### PR TITLE
chore: standardize npm script names

### PR DESCRIPTION
**Pull request by snitin315**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

Refers https://github.com/eslint/eslint/issues/14827

#### What changes did you make? (Give an overview)

This updates the names of the scripts in package.json to be consistent with the [new standard](https://eslint.org/docs/latest/contribute/package-json-conventions).

#### Is there anything you'd like reviewers to focus on?
Is there any script we need to update on Netlify?

### CODE REVIEW

**Review by amareshsm — DISMISSED** [review]
LGTM. Leaving this open for 2nd review.

**Review by kecrily — COMMENTED** [review]
It seems that no changes should be made to the indent style.

[The .editorconfig is used in eslint/eslint to apply 4 space indentation to all files](https://github.com/eslint/eslint/blob/a41a8e4a7da14726d6fce71a023f12101fd52fdb/.editorconfig#L4-L5). I think the eslint.org repository should be consistent as well, plus we could create a PR to add the missing .editorconfig file to it.

**Comment by harish-sethuraman** [comment]
@snitin315 can you fix the spacing so that we can review the PR?

**Comment by snitin315** [comment]
Yeah, my editor automatically formatted the indentation.

File: package.json [hunk 1]
```diff
@@ -14,30 +14,30 @@
     },
     "license": "Apache-2.0",
     "scripts": {
-        "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "install:playground": "cd src/playground && npm install --no-package-lock",
-        "watch:sass": "sass --watch src/assets/scss:src/assets/css",
-        "watch:eleventy": "cross-env IS_DEV=true eleventy --serve --port=2022",
-        "watch:webpack": "webpack watch --mode=development",
+        "build": "npm-run-all install:playground build:sass build:website build:playground build:minify-images",
+        "build:minify-images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
```
Comment (harish-sethuraman) [inline package.json:18]: Will confirm once with https://github.com/eslint/eslint/pull/17431#discussion_r1280829478 this thread and then make changes accordingly.

Comment (harish-sethuraman) [reply package.json:18]: Looks good 👍🏻

File: package.json [hunk 2]
```diff
@@ -14,30 +14,30 @@
...
+        "build:playground": "webpack --mode=production --progress",
+        "build:playground:watch": "webpack watch --mode=development",
         "build:sass": "sass --no-source-map --style=compressed src/assets/scss:src/assets/css",
-        "build:eleventy": "npx @11ty/eleventy",
-        "build:webpack": "webpack --mode=production --progress",
-        "fetch:team": "node tools/fetch-team-data.js",
+        "build:sass:watch": "sass --watch src/assets/scss:src/assets/css",
+        "build:website": "npx @11ty/eleventy",
+        "build:website:watch": "cross-env IS_DEV=true eleventy --serve --port=2022",
+        "fetch": "npm-run-all --parallel fetch:*",
```
Comment (harish-sethuraman) [inline package.json:26]: according to https://github.com/eslint/eslint/blob/main/docs/src/contribute/package-json-conventions.md we do not have any convention for fetch. May be we should consider adding it to the doc or replace these fetch with something like `build:fetch`?

Comment (kecrily) [reply package.json:26]: `build:fetch` looks good to me.

Comment (amareshsm) [reply package.json:26]: +1

Comment (nzakas) [reply package.json:26]: @mdjermanovic what do you think? I could live with adding a `fetch:` prefix.

Otherwise, maybe `build:remote-data`?

Comment (harish-sethuraman) [reply package.json:26]: Aligned on introducing `fetch:` scripts. Lets add it to the contribute's package-json-convention also @snitin315

Comment (snitin315) [reply package.json:26]: I'll update the docs.

**Review by harish-sethuraman — APPROVED** [review]
LGTM. make sure to update https://github.com/eslint/eslint/blob/main/docs/src/contribute/package-json-conventions.md as we mentioned

**Pull request merged**